### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by DanCarl857**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,520 @@
...
+    <!-- begin main -->
+    <main>
+        <div class="header-img-desc">
+            <img class="header" src="./assets/images/img-header.png" alt="img-header">
+            <img class="main-img" src="./assets/images/first.png" alt="" />
+            <p class="desc">Alan Guth was one of the first physicists to hypothesize the existence of inflation, which
+                explains how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8
+                billion years ago.<span class="credits">Rick Friedman for The New York Times</span></p>
+        </div>
+        <br>
```
Comment (maelfosso) [inline index.html:39]: No need to put that. If you want more space, you can use `<p>` or css properties like `margin` or `padding`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,520 @@
...
+            </div>
+            <div class="messages">
+            </div>
+            <!-- end editors pick -->
+        </div>
+    </main>
+    <!-- end main -->
+    <!-- begin theory of inflation -->
+
+    <div class="inflation">
```
Comment (maelfosso) [inline index.html]: That section should be inside the `<main>` tag. Please gently update it

File: index.html [hunk 3]
```diff
@@ -0,0 +1,520 @@
...
+            <div>
+                <hr>
+                <p class="popular-item">How the Raptors Won Their First N.B.A. Championship</p>
+            </div>
+        </div>
+        <div class="space"></div>
+    </div>
+    <!-- end inflation -->
+    <!-- begin footer -->
+    <nav class="nav">
```
Comment (maelfosso) [inline index.html]: No `<nav>` at that position. You should put it inside `<footer>`.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,520 @@
...
+        <a href="NY Logo">
+            <img src="assets/images/title.png" alt="The-New-York-Times-logo">
+        </a>
+        <a href="Go to Home Page">Go to Home Page »</a>
+        <div class="itemZ">
+            <div class="showTitle hideTitle">NEWS</div>
+            <div class="showTitle-a hideTitle-a"><a href="">NEWS</a></div>
+            <div class="showItems hideItems"></div>
```
Comment (maelfosso) [inline index.html]: Why did you put that line? It doesn't have any utility. Gently remove it, please.

**Comment by maelfosso** [comment]
Hello,

Awesome work !! You just have to fix some things

- [ ] Please gently correct problems highlighted by https://validator.w3.org

- [ ] The navigation bar should be fixed. Gently use `<nav>` into the header, directly nested.

- [ ] In the header use `display: flex`

- [ ] The NY Times logo is a link

- [ ] It should have a 'Read comments' button and social media icons

- [ ] Please, fontawsome is recommended for the icons. Use for icons at the navigation bar and social media icons

- [ ] Please, there is not the **Editor's Picks** aside and also the **Sign up for Science Times** section. Both must be floating. Gently put it, please.

- [ ] All images should be enclosed by a `<figure>` tag with `<figcaption>` that gives a description of the embedded image. Apply that everywhere there is an image

- [ ] Please, for `<nav>` at the end, inside the `<footer>`, it's highly preferable in that situation to use `display:flex`. Gently, update it, please.

Correct those points and submit a new code review request with the same pull request, please. No need to create a new one.

**Comment by maelfosso** [comment]
Hello,

- [ ] Please there is no **Read comments** button. Could you added it ?

- [ ] There is also no Floating section. The **Sign Up for Science Times** and **Editor's Picks** are not there. Gently added it, please.

- [ ] All images should be enclosed by a <figure> tag with <figcaption> that gives a description of the embedded image. You did it on images at the footer but not at the first large image at the beginning.

- [ ] `<div class="before-footer">` should be inside the `<footer>`. It's the beginning of the footer. Could you gently put it into the `<footer>`? Also, that `<div class="before-footer">` part, as it has been said up, should be implemented using `<nav>` tag. Correct it, please.

- [ ] `<div class="others">` and `<div class="inflation">` are both part of the `<main>`. Put them into it, please.

Correct those points and submit a new code review request please.

**Comment by Torres-ssf** [comment]
### Great work team
You worked a lot into this project and must be proud of it. You can merge to master now. Don't forget to submit the form

**Pull request merged**